drivers/usb/udc/udc_dwc2.c

```diff
@@ -1092,10 +1135,105 @@ static inline void dwc2_handle_oepint(const struct device *dev)
 	sys_write32(USB_DWC2_GINTSTS_OEPINT, (mem_addr_t)&base->gintsts);
 }
 
+static void dwc2_handle_incompisoin(const struct device *dev)
```

Comment:
Hehe, this name was not easy to read.

Reply:
I am sorry, but the name comes from Synopsys DWC2 otg databook.